### PR TITLE
Tribal Cookware Improvements

### PR DESCRIPTION
**Pull request by fludd12**
## About The Pull Request
Wooden bowls and a (significantly nerfed) butcher knife are now available for crafting without tools, meaning you can create the chopping block with tribal tools only.

Additionally, you can create a butter churn with 7 wood, letting you turn 15 milk into butter after a delay.

## Pre-Merge Checklist
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog
:cl:
add: Butter Churn - Craftable using wooden planks. Use a container of milk on it to churn butter!
add: Wooden Bowls - Craftable using wooden planks. A small reagent container that can be used to make soups or salads!
add: Carving Knife - Craftable in the crafting menu. A primitive and very dull butcher's cleaver.
balance: The wooden bowl and carving knife can be used to make the chopping block, meaning this is now easily available to tribals.
/:cl:

### CODE REVIEW

**Comment by fludd12** [comment]
thank you integration test

File: code/modules/farming/farming_structures.dm [hunk 1]
```diff
@@ -289,6 +289,47 @@
...
+	pass_flags = LETPASSTHROW
+	pass_flags_self = PASSTABLE | LETPASSTHROW
+	climbable = TRUE
+
+/obj/structure/butter_churn/attackby(obj/item/I, mob/user, params)
+	var/obj/item/reagent_containers/R = I
+	if(!istype(R))
+		return ..()
+	if(!(R.reagents.reagents_holder_flags & DRAINABLE))
+		to_chat(user, "<span class='warning'>You need to be able to pour \the [I] into \the [src].</span>")
```
Comment (Shadow-Quill) [inline code/modules/farming/farming_structures.dm:315]: `span_warning(...)` and `span_notice(...)`~

File: code/modules/mining/lavaland/ash_flora.dm [hunk 2]
```diff
@@ -224,6 +224,39 @@
...
+/obj/item/reagent_containers/glass/bowl/update_overlays()
+	. = ..()
+	if(reagents && reagents.total_volume)
+		. += mutable_appearance('icons/obj/lavaland/ash_flora.dmi', "woodenfullbowl", color = mix_color_from_reagents(reagents.reagent_list))
+
+/obj/item/reagent_containers/glass/bowl/mushroom_bowl/attackby(obj/item/I,mob/user, params)
+	if(istype(I, /obj/item/reagent_containers/food/snacks))
+		var/obj/item/reagent_containers/food/snacks/S = I
+		if(I.w_class > WEIGHT_CLASS_SMALL)
+			to_chat(user, "<span class='warning'>The ingredient is too big for [src]!</span>")
```
Comment (Shadow-Quill) [inline code/modules/mining/lavaland/ash_flora.dm:247]: `span_warning(...)`

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

**Comment by Shadow-Quill** [comment]
and all the applicable following ones

File: code/modules/mining/lavaland/ash_flora.dm [hunk 3]
```diff
@@ -224,6 +224,39 @@
 	else
 		. = ..()
 
+//COYOTE - Wooden Bowls
+/obj/item/reagent_containers/glass/bowl/wooden_bowl
```
Comment (GremlingSS) [inline code/modules/mining/lavaland/ash_flora.dm:228]: Should have just made this in the coyote_modular folder but its fine being here <3

**Review by GremlingSS — APPROVED** [review]
Gud to go, looks good!

**Pull request merged**